### DISCUSSION TITLE
404 when trying to login with Google OAuth

### DISCUSSION THREAD

**Discussion by bazluu** [pockethost/pockethost#326] [Q&A] [answered]
Trying to setup Google OAuth login on a Flutter project.
Upon trying to login I get a 404 from PocketHost: `Error during authentication: ClientException: {url: https://myinstance.pockethost.io/api/collections/users/auth-methods, isAbort: false, statusCode: 404, response: {code: 404, message: The requested resource wasn't found., data: {}}, originalError: null}`

This error is not present when using a local install of PocketBase.

**Comment by benallfree** [comment]
@LucaBazzea Since `https://myinstance.pockethost.io` isn't a real PocketHost instance, I can't help much. Also please consider jumping into Discord, you'll get faster responses from more people :)

**Reply by bazluu** [reply]
Will do, thanks

**Comment by bazluu** [comment, marked as answer]
Solved: I renamed the default collection `users` to `user` for aesthetic reasons 🙈
Reverting this change fixed the issue.
